package.json

```diff
@@ -53,8 +53,8 @@
     "brace": "^0.11.1",
     "crypto-browserify": "^3.12.0",
     "electron": "^13.1.4",
-    "ion-js": "^4.2.1",
-    "jsbi": "^3.1.6",
+    "ion-js": "4.1.0",
+    "jsbi": "^3.1.4",
```

Comment:
Downgraded it to keep the same version with console assets package, or we will get kind of `Argument of type [type] is not assignable to parameter of type [type]` issue for console. More specifically, this issue is from `node_modules/ion-js/dist/commonjs/es6/dom/Value Value[]`

Reply:
In this case we should probably have it as a peerDependency then, so that it uses the version defined by the parent application. We can also add it to devDependencies so that it installs automatically when running locally